### DISCUSSION TITLE
PrivacyItemController

### DISCUSSION THREAD

**Discussion by insproUPX** [MonwF/customiuizer#102] [Ideas]
Thank you.
Thanks to this module, I finally have a system that meets my requirements.
I miss one disable / enable switch of the green icon ( statusbar) that indicates the use of the camera and/or microphone 
(PrivacyItemController)
Now I'm doing it via the Magisk module:
Could this switch be included in the next version ?

**Comment by MonwF** [comment]
![Screenshot_2023-01-05-07-51-39-921_com miui securitycenter](https://user-images.githubusercontent.com/1267050/210671732-94efa4c3-3d26-4bd7-91b5-23bdf4f30d66.png)
from system settings
